services/ui-src/src/components/reports/ModalOverlayReportPage.test.tsx

```diff
@@ -74,6 +97,20 @@ describe("Test ModalOverlayReportPage (adding new program reporting information)
     await userEvent.click(closeButton);
     expect(screen.getByTestId("modal-overlay-report-page")).toBeVisible();
   });
+
+  it("State user should be able to delete existing entities", async () => {
+    // delete program entity
+    const closeButton = screen.getByAltText("delete icon");
```

Comment:
nit: we can use the getByRole to ensure element type. 
`const closeButton = screen.getByRole("button", { name: "delete icon" });`